Services/ArmouryService.cs

```diff
@@ -1,75 +1,36 @@
 ﻿using System.Text.Json;
 using Armours;
 using Monster_Builder;
+using Monster_Builder_Web_API.Repositories;
 
 namespace Monster_Builder_Web_API.Services
 {
     public class ArmouryService
     {
-        public Dictionary<string, Armour> armours;
         public Dictionary<string, Weapon> weapons;
 
-        public ArmouryService()
+        public ArmouryService(IArmourRepository armourRepository)
         {
-            armours = new Dictionary<string, Armour>();
+            _armourRepository = armourRepository;
             weapons = new Dictionary<string, Weapon>();
-            LoadBaseArmours();
             LoadBaseWeapons();
         }
-
```

Comment:
Separate concerns, this class should manage one of weapons or armour. It is often referred to as the single responsibility principle of [SOLID fame](https://www.freecodecamp.org/news/solid-principles-explained-in-plain-english/). It can be taken too far but this is a clear case of too many responsibilities. If you end up with some sort of inventory management then an inventory service may well call an armour and a weapons service but it will likely only use the add and remove, or maybe an update owner method on them and not do anything else.